# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/yabasic/hotfix-stdbool.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- yabasic-2.91.3/yabasic.h.vanilla	2025-06-22 22:31:26.830278372 +0200
+++ yabasic-2.91.3/yabasic.h	2025-06-22 22:31:36.137099699 +0200
@@ -98,6 +98,8 @@
 #define OPEN_PRINTER 8
 #define STDIO_STREAM 1234
 
+#include <stdbool.h>
+
 /* -------- variables and functions needed in all files and defined in ... -------- */
 
 /* main.c */
